bundles/org.openhab.binding.gce/README.md

```diff
@@ -0,0 +1,161 @@
+# GCE Binding
+
+This binding aims to handle various GCE Electronics equipments.
+IPX800 is a 8 relay webserver from gce-electronics with a lot of possibilities:
+
+* 8 Digital Input
+* 8 Relay (250v/ 10A / channel)
+* 4 Analog Input
+* 8 Counters
+* Ability to cascade up to 3 extensions for a total of 32 inputs / 32 relay
+
+Each IPX800 connected to openHAB must be configured with the setting 'Send data on status changed' on the website in M2M > TCP client.
+
+To make it simple, IPX800 is a simple device that drives output and retrieves input. 
+On input we generally connect push buttons (for instance house switchs), on ouputs we can connect light bulbs for instance.
+
+Features of the binding:
+
+ * Multi ipx support
+ * Direct TCP connection
+ * Auto reconnect
+ * Simple clic/Long press
+ * Pulse mode support
+
+## Binding Configuration
+
+There is no configuration at binding level.
+
+
+## Thing Configuration
+
+The IPX800v3 (ID : 'ipx800v3') accepts the following configuration parameters :
+
+| Property            | Default | Required | Description                 |
+|---------------------|---------|----------|-----------------------------|
+| hostname            |         | Yes      | IP address or hostname.     |
+| portNumber          | 9870    | No       | TCP client connection port. |
+| pullInterval*       | 5000    | No       | Refresh interval (in ms)    |
+
+The binding will query periodically the 'globalstatus.xml' page of the IPX to get fresh informations, this is
+especially usefull for Analog inputs and Counter as modification of these values on PLC side does not trigger any M2M message.
+
+The thing provides four groups of channels.
+
+### Digital Inputs
+
+This represents the inputs of the PLC. Each can be open or closed. They are usually commuted by physical devices like pushbuttons, magnets...
```

Comment:
Can you put sentences on a single line. It makes it easier to (future) review diffs.
```suggestion
This represents the inputs of the PLC. Each can be open or closed.
They are usually commuted by physical devices like pushbuttons, magnets...
```